.babelrc

```diff
@@ -1,3 +1,10 @@
 {
-  "presets": ["@babel/env"]
-}
\ No newline at end of file
+  "presets": [
```

Comment:
Not 100% sure why I needed to do this, but the ES6 style `import` statements were complaining about not being able to find a default export, until I made this change.